visualizer/actionButton.js

```diff
@@ -6,9 +6,29 @@
   if (typeof exports === 'object') {
     module.exports = initModule;
   } else {
-    initModule(root.ohmEditor);
+    initModule(root.ohmEditor, root.document);
   }
-})(this, function(ohmEditor) {
+})(this, function(ohmEditor, document) {
+  // DOM Helpers
+  // -----------
+
+  function createElement(sel, optContent) {
```

Comment:
Let's not duplicate this code. I pushed this function (and a few other DOM helpers) into a new module: domUtil.js.